applications/photo-asset-manager/SPECIFICATION.md

```diff
@@ -1,45 +1,51 @@
 # PAM tech spec
 
-This is the tech spec for the Photo Asset Management cross-service example. This doc details the specifics of the deliverable app.
+These are the technical specifications for the Photo Asset Management cross-service example, which was designed by the maintainers of this repository to showcase AWS services and SDKs.
 
-## Suggested Overview
-
-_Suggested overview for the per-language READMEs._
-
-The Photo Asset Management (PAM) example app uses Amazon Rekognition to categorize images, which are stored with Amazon S3 Intelligent-Tiering for cost savings. Users can upload new images. Those images are analyzed with label detection and the labels are stored in an Amazon DynamoDB table. Users can later request a bundle of images matching those labels. When images are requested, they will be retrieved from Amazon S3, zipped, and the user is sent a link to the zip.
+The Photo Asset Management (PAM) example app uses Amazon Rekognition to categorize images, which are stored with Amazon S3 Intelligent-Tiering for cost savings. Users can upload new images. Those images are analyzed with label detection and the labels are stored in an Amazon DynamoDB table. Users can later request a bundle of images matching those labels. When images are requested, they will be retrieved from Amazon S3, zipped, and sent to the user in the form of a link to the zipped file.
 
 ## Table of contents
-
-- Development / Deployment - Information on where to find AWS Cloud Development Kit (AWS CDK) deployment and development instructions.
-- Frontend - Describes the resources used to create the frontend, and how the frontend interacts with the backend.
-- Backend - Describes the resources used to create the backend.
-
-  - Common - Explains the concepts of common resources and language-specific resources. Also describes the common resources.
-  - Language-specific - Explains the programming language specific resources.
-
-- READMES - Defines the information needed for each language-specific readme.
-- Glossary - Lists common terms.
-- Appendices
-  - Appendix A - Policy details
-  - Appendix B - Lambda triggers
-
-<img width="4400" alt="PAM Diagram" src="https://user-images.githubusercontent.com/2723491/226400489-8ce85f78-fd53-4bcb-adda-42a230964a4c.png">
+- [High-level architecture](#high-level-architecture)
```

Comment:
I am generally not a fan of manual tables of contents in a markdown file. Entirely too much work to manage.

Reply:
Talked w/ team and we decided it was helpful. Let's remove it if it causes toil.